selenoid/docker.go

```diff
@@ -261,6 +261,7 @@ func (c *DockerConfigurator) getImage(name string, version string) *types.ImageS
 		c.Errorf("Failed to list images: %v", err)
 		return nil
 	}
+	//name = c.getFullyQualifiedImageRef(name)
```

Comment:
what it suppose to mean?

Reply:
Hmm, accidentally commented stuff. Need to recheck.